lib/block.rb

```diff
@@ -0,0 +1,18 @@
+require 'Date'
```

Comment:
When you require gems built into Ruby you should require them with their lowercase name:

```suggestion
require 'date'
```

(Using the `'Date'` won't work if you are using a computer that has case sensitive file names, which will be an issue once we start deploying projects.)